### DISCUSSION TITLE
Mapping a header and a sub-header to a title and a subtitle in powerpoint slide

### DISCUSSION THREAD

**Discussion by slavakx** [quarto-dev/quarto-cli#6452] [Q&A]
Description

Hi, 

I there any way to map a header and a sub-header to a title and a subtitle in powerpoint slide? 

For example: 

`##` Header
`###` Subheader 

I would expect to have a title and a subtitle in a slide. 

Thanks

**Comment by mcanouil** [comment]
No because that is not part of the template layout from Pandoc.
See <https://quarto.org/docs/presentations/powerpoint.html#powerpoint-templates>.

**Comment by slavakx** [comment]
I thought so but is it not a limitation that could be easily removed?
Also why not to introduce an attribute to explicitly define what layout
should be used for a current slide?

**Reply by mcanouil** [reply]
This is Pandoc limitation on the PowerPoint template.
The layout are limited to the one it knows it exist for sure: https://pandoc.org/MANUAL.html#powerpoint-layout-choice
